lib/arke/orderbook.rb

```diff
@@ -1,10 +1,38 @@
-# encoding: UTF-8
-# frozen_string_literal: true
-
-require 'rbtree'
-require_relative 'price_level'
-
 module Arke
   class Orderbook
+
+    attr_reader :book
+
+    def initialize(market)
+      @market = market
+      @book = {
+        sell: [],
+        buy: []
+      }
+    end
+
+    def add(order)
+      @book[order.side].push(order) unless @book[order.side].find {|b| b.id == order.id }
```

Comment:
everything after `unless` is `contains?` method code, just `unless contains?(order)` should work as well